working/templates/srmfunction/ProfileLoadScript_1/ProfileLoadScript_1.cs

```diff
@@ -66,6 +70,95 @@ public class Script
 	/// <param name="engine">Link with SLAutomation process.</param>
 	public void Run(Engine engine)
```

Comment:
Consider providing IEngine instead of Engine

Reply:
I will leave it as it is since the concrete Engine class is needed in the ProfileParameterEntryHelper class constructor.